TestYumConveror and TestYumPacker fail to find testdata

## Version of Singularity:

singularity version v3.0.0-alpha.2-222-gd338693

### Expected behavior

=== RUN   TestYumConveyor
--- PASS: TestYumConveyor (0.00s)
=== RUN   TestYumPacker
--- PASS: TestYumPacker (0.00s)

### Actual behavior

=== RUN   TestYumConveyor
--- FAIL: TestYumConveyor (0.00s)
    conveyorPacker_yum_test.go:34: unable to open file ./testdata_good/yum/yum: open ./testdata_good/yum/yum: no such file or directory
=== RUN   TestYumPacker
--- FAIL: TestYumPacker (0.00s)
    conveyorPacker_yum_test.go:70: unable to open file ./testdata_good/yum/yum: open ./testdata_good/yum/yum: no such file or directory

### Steps to reproduce behavior

Make test shows this as an error on my centos-7.0 VM.

--- a/src/pkg/build/sources/conveyorPacker_yum_test.go
+++ b/src/pkg/build/sources/conveyorPacker_yum_test.go
@@ -14,7 +14,7 @@ import (
 	"github.com/sylabs/singularity/src/pkg/test"
 )
 
-const yumDef = "./testdata_good/yum/yum"
+const yumDef = "../testdata_good/yum/yum"
 
 func TestYumConveyor(t *testing.T) {
 	if testing.Short() {

